.travis.yml

```diff
@@ -14,6 +14,6 @@ install:
   - pip install nose
 script:
   - nosetests notify_send_tests.py --with-coverage --cover-package=notify_send --cover-erase
-  - flake8 --ignore=E402 --max-line-length=100 notify_send.py notify_send_tests.py
+  - flake8 --ignore=E402,W504 --max-line-length=100 notify_send.py notify_send_tests.py
```

Comment:
I have fixed this in `master` (60c828717ce03da1a24537af261c0c3595ec8187) as Travis CI has been reporting this fail in there + this change is not specific to the present PR (this fix should have preferably been submitted in a separate PR, which I would have merged right away).